examples/examplevm/examplevm.h

```diff
@@ -0,0 +1,13 @@
+/* EVMC: Ethereum Client-VM Connector API.
+ * Copyright 2018 Pawel Bylica.
+ * Licensed under the MIT License. See the LICENSE file.
+ */
+
+#pragma once
```

Comment:
Is this a C or C++ header? :)

Reply:
C. I will have to fix comments syntax in evmc.h some day. GCC -pedantic is complaining.

Reply:
For me `#pragma once` is C++-ism.

Reply:
Maybe, but none of the compilers complains about it.

Reply:
I think in pedantic mode GCC would complain, since it is not part of <= C99 IIRC.

gcc and clang are too liberal allowing C++-ism in C by default.